packages/web-react/.eslintrc.js

```diff
@@ -0,0 +1,96 @@
+module.exports = {
+  root: true,
+  env: {
+    es6: true,
+    browser: true,
+  },
+
+  parser: '@typescript-eslint/parser', // the TypeScript parser we installed earlier
+
+  ignorePatterns: ['node_modules', '!.*.js'],
+
+  parserOptions: {
+    project: './config/tsconfig.eslint.json',
+    ecmaFeatures: { jsx: true }, // Allows for the parsing of JSX
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+  },
+
+  settings: {
+    'import/resolver': {
+      node: {
+        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+      },
+    },
+  },
+
+  extends: [
+    '@lmc-eu/eslint-config-base',
+    'prettier',
+    // "plugin:@typescript-eslint/eslint-recommended", // eslint TypeScript rules (github.com/typescript-eslint/typescript-eslint)
```

Comment:
Consider to remove those plugins. I'd rather prefer to have a weak check now and this solve with another CQT package for typescript.

Reply:
Can we remove the lines that are commented out? Or are they here to be uncommented soon?

Reply:
Yes, I was waiting for an opinion.